protocol/avdl/keybase1/teams.avdl

```diff
@@ -646,6 +646,9 @@ protocol teams {
     boolean open;
     TeamRole openTeamJoinAs;
 
+    // restricted-bot configurations
+    map<UserVersion, TeamBotSettings> bots;
```

Comment:
Where `B` is this commit and `A` is the previous commit. Remember to handle a client running `A` that loads a team made after `B`. And then upgrades to `B` without doing db nuke. Might need to bust cache or backfill.

Reply:
TODO: check an old client loading a team with the new link. Should error out so we'll have to have a few releases with reader support before we have these links in the wild